paging/paging-common/src/test/kotlin/androidx/paging/PagingDataDifferTest.kt

```diff
@@ -1577,10 +1922,28 @@ class PagingDataDifferTest(
                     ).also { pagingSources.add(it) }
                 }
             ),
-        block: (SimpleDiffer, TestDispatcher, MutableList<TestPagingSource>) -> Unit
+        block: (
+            SimpleDiffer,
+            TestDispatcher,
+            MutableList<TestPagingSource>,
```

Comment:
This and the other `MutableList` parameters can be exposed as a `List`. I don't think we need any `block` to be able to modify these lists :)

Reply:
done